python/shotgun_desktop/startup.py

```diff
@@ -535,13 +534,33 @@ def __post_bootstrap_engine(splash, app_bootstrap, engine):
 
     startup_version = startup_desc.version
 
-    return engine.run(
-        splash,
-        version=app_bootstrap.get_version(),
-        startup_version=startup_version,
-        startup_descriptor=startup_desc
-    )
-
+    # If the site config is running an older version of the desktop engine, it
+    # doesn't include browser integration, so we'll launch it ourselves.
+    server = None
+    if LooseVersion(engine.version) < "v2.1.0":
+        logger.debug("Old Desktop engine detected, launching legacy websocket server.")
```

Comment:
i suggest include the version number of the desktop engine in this debug message.